content/developer/reference/javascript/generic_components.rst

```diff
@@ -44,4 +60,294 @@ Props
       - `boolean`
       - if true, the checkbox is disabled, otherwise it is enabled
 
+.. _generic_components/dropdown:
+
+Dropdown
+========
+
+Location
+--------
+
+`@web/core/dropdown/dropdown` and `@web/core/dropdown/dropdown_item`  
+
+Description
+-----------
+
+Dropdowns are surprisingly complicated components. They need to provide many
+features such as:
+
+- Toggle the item list on click
+- Direct siblings dropdowns: when one is open, toggle others on hover
+- Close on outside click
+- Close the item list when an item is selected
+- Emits an event to inform which list item is clicked
+- supports sub dropdowns, up to any level
+- SIY: style it yourself
+- Configurable hotkey to open/close a dropdown or select a dropdown item
+- Keyboard navigation (arrows, enter...)
+- Reposition itself whenever the page scrolls or is resized
+- Smartly chose the direction it should open
+
+To solve these issues once and for all, the Odoo framework provides a set of two
+components: a `Dropdown` component (the actual dropdown), and `DropdownItem`,
+for each element in the item list.
+
+.. code-block:: xml
+
+   <Dropdown>
+       <!-- "default" slot content is rendered inside a button, by default -->
```

Comment:
it's the opposite now, the default slot is the items, and there is a toggler slot for the toggler (which is put inside a <button/> node). Also, there's a `toggler` props, and if it is set to `parent`, the dropdown doesn't render any toggler, as the toggling feature is handled by the parent node (use case: pivot cells)